sale_automatic_workflow/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Sale Automatic Workflow
+=======================
+
+Create workflows with more or less automatization and apply it on sales
+orders.
+
+A workflow can:
+
+- Apply default values:
+
+  * Packing Policy (partial, complete)
+  * Shipping Policy (prepaid, manual, postpaid, picking)
+  * Invoice On (ordered quantities, shipped quantities)
+  * Set the invoice's date to the sale order's date
+  * Set a sales team
+
+- Apply automatic actions:
+
+  * Validate the order (only if paid, always, never)
+  * Create an invoice
+  * Validate the invoice
+  * Confirm the picking
+
+This module is used by Magentoerpconnect and Prestashoperpconnect.
+It is well suited for other E-Commerce connectors as well.
+
+Usage
+=====
+
+To use this module, you need to:
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/sale-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```

Comment:
from [here](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst):
`help us smash it by providing detailed and welcomed feedback.`